zds/notification/receivers.py

```diff
@@ -284,11 +284,11 @@ def answer_comment_event(sender, **kwargs):
     assert user is not None
     # avoids creation of a ping if forum is not visible.
     if sender == Post:
-        if Topic.objects.filter(pk=comment.topic.forum.pk, group__in=list(user.groups.all)).first() is None:
+        if Topic.objects.filter(pk=comment.topic.forum.pk, group__in=list(user.groups.all)).exists():
```

Comment:
Je crois que tu as inversé la condition ici? Tu passes de `if X.first() is None:` à `if X.exists():`